app/views/layouts/courses.html.erb

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
Maybe I'm missing something but how is this different from assignment_content.html.erb?

Reply:
No, you're correct - this did end up being exactly the same as assignment_content.html.erb (barring a couple of lines in different places). I was going to suggest renaming assignment_content to something more general, but it looks like assignment_content is used as a layout in a bunch of places that aren't strictly assignments (e.g. section controller), so for now I suppose we can simply use assignment_content for courses as well and remove courses.html.erb